### PR TITLE
Tinfoil hats deconvert gangs

### PR DESCRIPTION
**Pull request by Ohagi-Chan**
## About The Pull Request
At the request of Psychenurse, who wanted a layman's way to deconvert gangs, that didn't require metagaming, head staff intervention or rely on cargo, I have made a change to tinfoil hats so that they can deconvert gangsters.

## Why It's Good For The Game
Gives crewmen a ghetto way to deconvert, making headless shifts less one-sided.

## Changelog
:cl:
balance: tinfoil hats deconvert gangs in addition to preventing gang mind control
/:cl:

### CODE REVIEW

File: code/modules/clothing/head/misc_special.dm [hunk 1]
```diff
@@ -307,7 +307,7 @@
 	equip_delay_other = 140
 	var/datum/brain_trauma/mild/phobia/conspiracies/paranoia
 
-/obj/item/clothing/head/foilhat/equipped(mob/living/carbon/human/user, slot)
+/*  austation begin /obj/item/clothing/head/foilhat/equipped(mob/living/carbon/human/user, slot)
```
Comment (TheFakeElon) [inline code/modules/clothing/head/misc_special.dm:310]: put the comment above the object

Comment (Ohagi-Chan) [reply code/modules/clothing/head/misc_special.dm:310]: the WHOLE object?

I'm replacing the line here.
my /austation folder now has the proc.

Comment (Ohagi-Chan) [reply code/modules/clothing/head/misc_special.dm:310]: oh, I see

**Comment by TheFakeElon** [comment]
the problem I see with this is it's not really a "ghetto deconversion" when it's 100x better in every single way, one (very cheap) hat has unlimited deconversions and can be mass produced. Not to mention hat throwing that will let you instantly deconvert people from a distance

**Comment by Ohagi-Chan** [comment]
> the problem I see with this is it's not really a "ghetto deconversion" when it's 100x better in every single way, one (very cheap) hat has unlimited deconversions and can be mass produced. Not to mention hat throwing that will let you instantly deconvert people from a distance

I've put this up on github because I had already completed this solution at the time.
When I can be bothered, I will be putting up an alternative that allows the implanter chair to be built (it's in the code but doesn't have a way to build it other than adminspawn).

I'd honestly recommend the implant chair to this solution.

**Comment by TheFakeElon** [comment]
correction: tinfoil hats can't be thrown

**Comment by TheFakeElon** [comment]
why didn't you just PR the implanter chair then ;)

**Comment by YakiAttaki** [comment]
por que no los dos?

**Comment by Ohagi-Chan** [comment]
> why didn't you just PR the implanter chair then ;)

Because at the time of discovering implanter chairs, I had already finished making this code.

**Review by MCterra10 — CHANGES_REQUESTED** [review]
I'm not a fan.
Maybe as an alternative you could copy the revolution code so that enough bashing to the head causes deconversion?

**Comment by MCterra10** [comment]
Closing, haven't heard back.

**Comment by Ohagi-Chan** [comment]
I'm not even mad.
bad idea in the first place.

**Pull request closed without merging**